docs/codeql/codeql-language-guides/codeql-for-java.rst

```diff
@@ -1,9 +1,16 @@
 .. _codeql-for-java:
 
-CodeQL for Java
-===============
+CodeQL for Java and Kotlin
+==========================
 
-Experiment and learn how to write effective and efficient queries for CodeQL databases generated from Java codebases.
+Experiment and learn how to write effective and efficient queries for CodeQL databases generated from Java and Kotlin codebases.
+
+.. include:: ../reusables/kotlin-beta-note.rst
+
+
+.. pull-quote:: Enabling Kotlin support
+
+   To enable Kotlin support, you should enable ``java`` as a language.
```

Comment:
I think it would be worth expanding on this, e.g.:
```
CodeQL treats Java and Kotlin as parts of the same language, so to enable Kotlin support you should enable ``java`` as a language.

Reply:
Thanks @igfoo I've added this and your other comment in a single commit 👍